mozilla/foundation.mozilla.org
# updated to synchronized field, also updated template to pull localized version of signup

# Description

This PR updates the 'signup' field from TranslatableField to SynchronizedField, as changes to the signup field on any English mozfest page would not get synchronized to its localized versions.

Link to sample test page: 
http://mozfest.localhost:8000/en/proposals/
http://mozfest.localhost:8000/fr/proposals/

Related PRs/issues: #9642

# Steps to test

1. Check this branch out and run `inv copy-prod-db` **Note**: This command can take a while!
2. Navigate to http://mozfest.localhost:8000/en/proposals/ and note that there is no "signup" set.
<img width="500" alt="Screen Shot 2022-11-08 at 2 36 02 PM" src="https://user-images.githubusercontent.com/18314510/200691448-fc6e8a0b-26ef-40b6-91ad-02a98db3ab11.png">

3. Navigate to http://mozfest.localhost:8000/fr/proposals/ and note that there is an existing "signup" form.
<img width="500" alt="Screen Shot 2022-11-08 at 2 36 02 PM" src="https://user-images.githubusercontent.com/18314510/200692045-c7baac6e-83d6-4a1a-adb2-7663be5bbccd.png">

4. Log into the CMS http://mozfest.localhost:8000/ , navigate to the "Call to proposals" page, and select "Sync Translations".
<img width="700" alt="Screen Shot 2022-11-08 at 2 36 02 PM" src="https://user-images.githubusercontent.com/18314510/200691410-cf99af57-d70f-4ab6-8756-682e25ad2747.png">

5. Go back to French version of the page http://mozfest.localhost:8000/en/proposals/ , and note that it is now the same as the English version. (No "signup" set). 
**Side note:** It appears that the use_wide_template field is also not being synced among translations, this will be taken care of in #9645
<img width="500" alt="Screen Shot 2022-11-08 at 2 36 02 PM" src="https://user-images.githubusercontent.com/18314510/200691915-69738e31-4201-43e3-9352-c16d028562af.png">

6. To test further, make a change to "signup" field of the English version of the page, publish, and click "Sync Translations".
7. Verify that the change took effect in both the English and French versions of the page. If everything is looking as expected, testing is complete!

# Checklist



**Tests**

**Changes in Models:**
- ~~ ] Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?~~

**Documentation:**
